components/shared/components/Widget/components/panes/PaymentPane/PaymentPane.tsx

```diff
@@ -6,25 +6,20 @@ import {
   WidgetPane3BankRecurringProps,
   WidgetPane3BankSingleProps,
   WidgetPane3ReferralsProps,
-  WidgetPane3VippsRecurringProps,
-  WidgetPane3VippsSingleProps,
 } from "../../../types/WidgetProps";
-import { Pane } from "../Panes.style";
 import { ResultPane } from "./Bank/ResultPane";
 import { VippsPane } from "./Vipps/VippsPane";
+import { VippsProps } from "../../../types/VippsProps";
 
 export const PaymentPane: React.FC<{
-  text: WidgetPane3BankRecurringProps &
-    WidgetPane3BankSingleProps &
-    WidgetPane3VippsRecurringProps &
-    WidgetPane3VippsSingleProps &
-    WidgetPane3ReferralsProps;
-}> = ({ text }) => {
+  text: WidgetPane3BankRecurringProps & WidgetPane3BankSingleProps & WidgetPane3ReferralsProps;
+  vipps?: VippsProps;
+}> = ({ text, vipps }) => {
   const method = useSelector((state: State) => state.donation.method);
 
-  return (
-    <Pane>
-      {method === PaymentMethod.BANK && (
+  switch (method) {
```

Comment:
I really like switches, they feel semantically very clear to me 👍 

What happened to the wrapping `<Pane>` element though?

Reply:
Nvm, removal of `<Pane>` is fine, as mentioned in another comment 👍